### ISSUE TITLE
Unable to find the documentation for cloud deployment

### ISSUE THREAD

**Issue by syedmbzaidi** [technologiestiftung/giessdenkiez-de-dwd-harvester#37]
Hi Team,
I am implementing the same project for the City of Erfurt. Currently, I am trying to deploy the backend (fargate cluster). But I am unable to do so. I want to ask how you deploy the code on the aws cloud? 

I didn't find any ecs-cli or aws-cli commands in the repository. Please guide me in the right direction. I am trying to deploy the app on aws directly. I have a postgres db ready. But I am unable to find the documentation for the deployment over the cloud.

Regards
Bilal Zaidi

**Comment by ff6347** [comment]
Hi,

We are using terraform to deploy to fargate and to provision the S3. See terraform/ecs-harvester in this repo. 
But actually having GitHub action would be enough in this case.

**Comment by syedmbzaidi** [comment]
Hi, 
Thanks for the reply. I have deployed the backend using terraform scripts. But the ECS task is not able to connect with databases. I have Terraform 0.13.5 . If I create a new task it ends and the following error can be found in cloudwatch.

![image](https://user-images.githubusercontent.com/73298336/99900746-ad1b1980-2cb1-11eb-8234-fd7074eb6fb7.png)

**Comment by ff6347** [comment]
Does the DB exist?
Are they in the same VPC?
Is the DB ingress restricted?
Can you run the harvester.py from your local machine?

AWS config is kind of black magic. Maybe running it from GitHub actions is easier.

**Comment by syedmbzaidi** [comment]
Yes, It was the issue because the database was not created by terraform. while debugging I was able to create the DB. 
Thank you so much for the response.

**Comment by ff6347** [comment]
@syedmbzaidi You are welcome. The documentation is not that great. Shame on me. The schema for the DB can be found in [here](https://github.com/technologiestiftung/giessdenkiez-de-postgres-api/tree/master/test/sql)  which might be a little outdated but still should work. We added some extra columns there are pretty berlin specific.
